lib/elixir/lib/file.ex

```diff
@@ -670,8 +670,9 @@ defmodule File do
       end
 
   """
-  @spec cp_r(Path.t, Path.t, (Path.t, Path.t -> boolean)) :: {:ok, [binary]} | {:error, posix, binary}
-  def cp_r(source, destination, callback \\ fn(_, _) -> true end) when is_function(callback) do
+  @spec cp_r(Path.t, Path.t, (Path.t, Path.t -> boolean)) ::
+        {:ok, [binary]} | {:error, posix, binary}
+  def cp_r(source, destination, callback \\ fn(_, _) -> true end) when is_function(callback, 2) do
```

Comment:
I see that the guard is also being changed but I think the newline should stay as-is.